api/settings.js

```diff
@@ -19,7 +19,8 @@ const AccountRootFlags = {
   DisallowXRP:     { name:  'disallow_xrp', value: 0x00080000 },
   DisableMaster:   { name:  'disable_master', value: 0x00100000 },
   NoFreeze:        { name:  'no_freeze', value: 0x00200000 },
-  GlobalFreeze:    { name:  'global_freeze', value: 0x00400000 }
+  GlobalFreeze:    { name:  'global_freeze', value: 0x00400000 },
+  DefaultRipple:   { name:  'default_ripple', value: ripple.Remote.flags.account_root.DefaultRipple }
```

Comment:
I just made the changes you suggested, and found that two of the flags were missing from ripple-lib https://github.com/ripple/ripple-lib/pull/314